crates/bin/prove_block/tests/prove_block.rs

```diff
@@ -1,7 +1,6 @@
 use cairo_vm::types::layout_name::LayoutName;
-use cairo_vm::types::relocatable::MaybeRelocatable;
 use cairo_vm::vm::runners::cairo_pie::CairoPie;
-use prove_block::{debug_prove_error, prove_block};
+use prove_block::{debug_prove_error, get_memory_segment, prove_block};
 use rstest::rstest;
 
 pub const DEFAULT_COMPILED_OS: &[u8] = include_bytes!("../../../../build/os_latest.json");
```

Comment:
super picky but.. since you've moved `DEFAULT_COMPILED_OS`, you can remove this constant and use the one that is in lib.rs

Reply:
Totally right. Good catch :)